lib/CsvToMarkdown.js

```diff
@@ -20,7 +20,8 @@ function csvToMarkdown(csvContent, delimiter, hasHeader) {
         if (typeof tabularData[i] == "undefined") {
             tabularData[i] = [];
         }
-        var row = e.split(delimiter);
+        var regex = new RegExp('(?!\\B"[^"]*)' + delimiter + '(?![^"]*"\\B)');
```

Comment:
In trying to fully understand this regex, as well as playing with it here https://regex101.com/r/9TKbIp/1/

I'm not certain the first negative lookahead actually does anything at all?

WIthout it https://regex101.com/r/9TKbIp/2 I get the same number of matches here, and I can't seem to find a way to make the result different… 

If I'm reading the first regex correctly it's simply negatively looking ahead at the delimiter itself which is unessessary? Let me know if I am mistaken.